llvm/test/TableGen/PPCGenRegisterBank.inc

```diff
@@ -0,0 +1,176 @@
+/*===- TableGen'erated file -------------------------------------*- C++ -*-===*\
```

Comment:
This is just a generated output, it's not a test. The test needs to be the tablegen input checking the generated output

Reply:
For sure, that was just to show the generated code as requested above.